sources/packages/web/src/components/common/modals/UserNoteConfirmModal.vue

```diff
@@ -80,14 +80,16 @@ export default defineComponent({
       required: false,
       default: true,
     },
-    loading: {
-      type: Boolean,
-      default: false,
+    notesLabel: {
+      type: String,
+      required: true,
```

Comment:
required can be false.

Reply:
Removed since it is redundant to keep it as false.